README.md

```diff
@@ -17,6 +17,16 @@ BATON_API_TOKEN=oktaAPIToken BATON_DOMAIN=domain-1234.okta.com baton-okta
 baton resources
 ```
 
+## Applications
+
+```
+BATON_OKTA_CLIENT_ID=appClientID \
+BATON_OKTA_PRIVATE_KEY='auth0.pem' \
```

Comment:
The key isn't in pem format. The file name should be auth.key or something like that.